Assets/LeapMotionModules/VRVisualizer/ScenesResources/Scripts/VisualizerManager.cs

```diff
@@ -94,6 +97,21 @@ void Update()
             m_controller.SetPolicy(Controller.PolicyFlag.POLICY_OPTIMIZE_HMD);
         }
       }
+      //update render frame display
+      m_deltaTime += (Time.deltaTime - m_deltaTime) * 0.1f;
```

Comment:
We should be using SmoothedFloat for all smoothing related needs, since SmoothedFloat is framerate independent, while traditional exponential smoothing is not.